ui/src/app/components/help-sidebar.spec.tsx

```diff
@@ -701,12 +700,12 @@ describe('HelpSidebar', () => {
 
     expect(wrapper.text()).not.toContain('RStudio Cloud Environment');
 
-    const cromwellIcon = wrapper.find({
```

Comment:
`cromwellIcon` => `rstudioIcon` since this is an RStudio test. This is unrelated to the PR.